provider/softlayer/vendor/vendor.json

```diff
@@ -1,19 +1,79 @@
 {
 	"comment": "",
-	"ignore": "test",
```

Comment:
Do you mind formatting this file? I believe this was previously done manually. We don't have a Makefile or similar on this project, so there's no obvious way of knowing it, but should clean up the diff.

```
$ go get -u github.com/magiconair/vendorfmt/cmd/vendorfmt
$ vendorfmt
```

Reply:
I'm also wondering if we can drop the softlayer updates here too.

Reply:
Formatted. We have to keep the updates. The ability to set a custom user agent wasn't available in the vendored version.

Reply:
@sethvargo Given softlayer isn't one of the providers we run regular acceptance tests against I'd rather drop these changes to it -- it is ok if it does not send the user agent at this time.